owslib/ogcapi/processes.py

```diff
@@ -83,15 +100,28 @@ def execute(self, process_id: str, json: dict) -> dict:
         """
         # validate input
         m.Execute.parse_obj(json)
-
         path = f'processes/{process_id}/jobs'
         resp = self._request_post(path, json)
+
+        # Carsten, is it a server non-conformity that we can't just return m.Result.parse_obj(resp.json()) ?
```

Comment:
I was unsure were the `location` parameter is. On weaver it is returned via the header ... but it should also be the json.